service/README.md

```diff
@@ -53,17 +53,87 @@ Change the variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/configurer/utils/fp
 the service.
 
 ### Run the application (from IntelliiJ)
- 
-Ensure the Spring Boot is started with local profile 
+
+Ensure the Spring Boot is started with local profile
```

Comment:
We also need the user-mappings profile and the feature-toggle profile

Reply:
Done.

Reply:
Should it also be mentioned that the user-mappings is generated by the `add-users` script and that a member of the team needs to be contacted for the details of the feature-toggle profile?

Reply:
Hmm not sure, it's a bit out of scope changes I am really documenting, and we already have mentioned we need to run add-users it the main README  🤔 
I mean ultimately I want to documents mappings as well, where I will add more details in.